src/components/FileUpload.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react'
+import config from 'config/configFromCookie'
+import 'styles/qtcstyles.scss'
```

Comment:
An update from develop will change this but I believe it is now called just `styles`
```suggestion
import 'styles/styles.scss'
```